horace_core/sqw/file_io/@horace_binfile_interface/deactivate.m

```diff
@@ -5,6 +5,7 @@
 % To use for MPI transfers between workers when open file can
 % not be transferred between workers but everything else can
 if ~isempty(obj.file_closer_)
+    obj.file_closer_.delete();
     obj.file_closer_ = [];
 end
 obj = obj.fclose();
```

Comment:
Is this also overkill?

Reply:
well, it certainly fixes unit tests failing before. But it certainly can be done more efficiently #1322.

Reply:
@abuts Again, put the ticket number into comment in the file so we know why your fix is there

Reply:
see above